tutorials/quickstart/tutorial-reliability.md

```diff
@@ -111,9 +110,9 @@ The purpose of this app is to host different tools within a sandbox environment,
 
 ![Service Pulse: Dashboard View](pulse-dashboard.png)
 
-The screenshot shows how ServicePulse monitors the operational health of your system. It tracks **Heartbeats** from your messaging endpoints, ensuring that they are running and able to send messages. It tracks **Failed Messages** and allows you to retry them. It also supports **Custom Checks** allowing you to write code that checks the health of your external dependencies (such as connectivity to a web service or FTP server) so you can get a better idea of the overall health of your system.
+The screenshot shows how ServicePulse monitors the operational health of your system. It tracks **Heartbeats** from your messaging endpoints, ensuring that they are running and able to send messages. It tracks **Failed Messages** and allows you to retry them. It also supports **Custom Checks** allowing you to write code that verifies the health of your external dependencies (such as connectivity to a web service or FTP server) so you can get a better idea of the overall health of your system.
```

Comment:
Are you changing this because it feels redundant? I'm leaning more toward using "checks", because rather than redundant, it sort of affirms we named "Custom Checks" correctly and simply, does that make sense? Otherwise, I'd wonder why we didn't name it "Customer Verification" :P

Reply:
Interesting take, yes, I changed because it sounded redundant, but I can change it back.